receivers/telegram/telegram.go

```diff
@@ -116,7 +123,18 @@ func (tn *Notifier) buildTelegramMessage(ctx context.Context, as []*types.Alert)
 
 	tmpl, _ := templates.TmplText(ctx, tn.tmpl, as, tn.log, &tmplErr)
 	// Telegram supports 4096 chars max
-	messageText, truncated := receivers.TruncateInRunes(tmpl(tn.settings.Message), telegramMaxMessageLenRunes)
+	rawMessage := tmpl(tn.settings.Message)
+	if tn.settings.IncludeScreenshotURL {
+		_ = images.WithStoredImages(ctx, tn.log, tn.images, func(_ int, image images.Image) error {
+			// Change to add the template ImageURL
+			imageURL, err := url.Parse(image.URL)
+			if err == nil {
+				tn.tmpl.ExternalURL = imageURL
```

Comment:
This will not work because the ExternalURL is used in other places. We need to figure out a better place for this URL. I would add it to annotations under "Screenshot URL".

Reply:
I configured my local instance to store screenshot in cloud. That's what I get 
![image](https://github.com/user-attachments/assets/f2ee4ebe-01ba-4bb9-a190-9347d94fa40f)

All urls are replaced with screenshot URL which is not correct.